modules/table/src/integrationTest/java/org/apache/ignite/distributed/ItTxDistributedTestSingleNode.java

```diff
@@ -635,9 +643,9 @@ protected boolean assertPartitionsSame(TableImpl table, int partId) {
 
             MvPartitionStorage storage = listener.getMvStorage();
 
-            if (hash == 0) {
-                hash = storage.hashCode();
-            } else if (hash != storage.hashCode()) {
+            if (storageIdx == 0) {
```

Comment:
Please add todo here that we need to change the way how we compare partitions after consistency checker feature will be implemented https://issues.apache.org/jira/browse/IGNITE-18869

Reply:
I added the todo in a summary of the method.